docs/aws.md

```diff
@@ -214,6 +214,8 @@ aws {
 
 Different queues bound to the same or different Compute Environments can be configured according to each process' requirements.
 
+Nextflow on AWS Batch supports only a single queue in your pipeline configuration. Multiple queues are not supported.
+
```

Comment:
```suggestion
```

We need to correct that's possible to provide multiple queues in the same settings, that's a very specific exception